internal/cmdrpkgapprove/command.go

```diff
@@ -86,15 +89,25 @@ func (r *runner) preRunE(cmd *cobra.Command, args []string) error {
 
 func (r *runner) runE(cmd *cobra.Command, args []string) error {
 	const op errors.Op = command + ".runE"
+	var messages []string
 
 	namespace := *r.cfg.Namespace
-	name := args[0]
 
-	if err := porch.UpdatePackageRevisionApproval(r.ctx, r.client, client.ObjectKey{
-		Namespace: namespace,
-		Name:      name,
-	}, v1alpha1.PackageRevisionLifecycleFinal); err != nil {
-		return errors.E(op, err)
+	for _, name := range args {
+		switch err := porch.UpdatePackageRevisionApproval(r.ctx, r.client, client.ObjectKey{
```

Comment:
qq: just curious about the switch/case usage here. Does it infer `UpdatePackageRevisionApproval` requires  other error handlers in the future?  If not, if/else seems to be more readable.